ztp/policygenerator/policyGen/policyHelper.go

```diff
@@ -58,3 +58,35 @@ func CheckNameLength(namespace string, name string) error {
 	}
 	return nil
 }
+
+var (
+	// Singleton template for ObjectTemplates
+	refObjTemplate *utils.ObjectTemplates
+)
+
+// Create a new ObjectTemplate for the given resource with values
+// populated from the reference ACM template (pulled from
+// policyBuilder)
+func BuildObjectTemplateFor(pbuilder *PolicyBuilder, resource generatedCR) (utils.ObjectTemplates, error) {
```

Comment:
You don't really need to pass pbuilder as parameter here just pass the ComplianceType & resource. Keep the validation for default value out side.

Reply:
Leftover from previous fix. I'll remove it.